src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,19 @@
 # Release Notes
 
+## July 26, 2023
+
+### Android Core 2.3.0
+
+* Added the DeviceInforming.getSystemLocale() API, which returns the locale set by the user in the system.
+* Implemented database recovery in SQLiteDataQueue for cases where adding or removing an entry fails due to unexpected errors.
+* Fixed an issue with the rounded corners and placement of in-app messages on Android 21/22.
+* Fixed a bug where the cached appId could overwrite the one set with MobileCore.configureWithAppId() before extension registration.
+
+### Android Lifecycle 2.0.4
+
+* Lifecycle extension now includes the system locale, indicating the device's preferred locale, in its published shared state.
+* Lifecycle launch events will now include the application._dc.language XDM field, which signifies the device's preferred locale. 
```

Comment:
- use present tense: launch events now include ... (same in the general notes)
- can you update the lifecycle metrics page too with the new field - https://developer.adobe.com/client-sdks/documentation/mobile-core/lifecycle/metrics/#lifecycle-application-foreground-metrics